public/js/selfoss-ui.js

```diff
@@ -35,22 +35,28 @@ selfoss.ui = {
     },
 
 
-    refreshItemStatuses: function(entry_statuses) {
+    refreshItemStatuses: function(entryStatuses) {
         $('.entry').each(function(index, item) {
             var id = $(this).data(('entry-id'));
-            new_status = entry_statuses.find(function(entry_status) {
-                return entry_status.id == id;
+            var newStatus = false;
+            entryStatuses.some(function(entryStatus) {
+                if( entryStatus.id == id )
+                    newStatus = entryStatus.id;
```

Comment:
Should not this be just `newStatus = entryStatus`? And since you are actually trying to **find** some `entryStatus`, why do you use `Array.prototype.some`?